crates/trie/trie/src/proof_v2/mod.rs

```diff
@@ -303,8 +390,7 @@ where
         leaf_val: VE::DeferredEncoder,
     ) -> Result<(), StateProofError> {
         // Before pushing the new leaf onto the `child_stack` we need to commit the previous last
-        // child (ie the first child of this new branch), so that only `child_stack`'s final child
-        // is a non-RlpNode.
+        // child, so that only `child_stack`'s final child is a non-RlpNode.
         self.commit_last_child(targets)?;
 
         // Once the first child is committed we set the new child's bit on the top branch's
```

Comment:
should this be first child or is it last child?


i,e
`self.commit_last_child(targets)?;`